README.md

```diff
@@ -1,6 +1,9 @@
 Shx
 ===
 
+[![Travis](https://img.shields.io/travis/shelljs/shx.svg)](https://travis-ci.org/shelljs/shx)
+[![AppVeyor](https://img.shields.io/appveyor/ci/ariporad/shx.svg?maxAge=2592000)](https://ci.appveyor.com/project/ariporad/shx)
```

Comment:
I don't know why this is `ariporad/shx` instead of `shelljs/shx` I've confirmed that it's the actual repo under @shelljs, not my fork. But this is the only thing that works. Go figure.